tests/fsharp/core/csext/test.fsx

```diff
@@ -311,6 +311,9 @@ module TupleSRTP =
     let v1b =  (^T : (member get_Item2 : unit -> _ ) (new System.Tuple<int,int>(1,3)))
     let v2b =  (^T : (member get_Item2 : unit -> _ ) (System.Tuple<int,int>(1,3)))
     let v3b =  (^T : (member get_Item2 : unit -> _ ) ((1,3)))
+    
+    let areEqualT8 = System.Tuple<_,_,_,_,_,_,_,_>(1,2,3,4,5,6,7,System.Tuple<_>  (8)  ) = (1,2,3,4,5,6,7,8)
```

Comment:
Will this also compile for:

```
     let areEqualT9 = System.Tuple<_,_,_,_,_,_,_,(int * int)>(1,2,3,4,5,6,7,(8,9)) = (1,2,3,4,5,6,7,8,9) 
```

Reply:
Yes, I added the 8-uple case as it was trickier to get it right, but it should work for any tuple size.

Reply:
I just added a commit that will allow that.